db/structs.go

```diff
@@ -973,6 +973,30 @@ type Tickets struct {
 	UpdatedAt    time.Time         `gorm:"type:timestamp;default:current_timestamp" json:"updated_at"`
 }
 
+type BroadcastType string
+
+const (
+	PoolBroadcast BroadcastType = "pool"
+
+	DirectBroadcast BroadcastType = "direct"
+)
+
+type ActionType string
+
+const (
+	ProcessAction ActionType = "process"
+
+	MessageAction ActionType = "message"
+)
+
+type TicketMessage struct {
+	BroadcastType   BroadcastType `gorm:"type:varchar(50)" json:"broadcastType"`
```

Comment:
@MahtabBukhari - We won’t need gorm on the ticketMessage struct - this is for passing around websocket messages in a structured format, but there is no associated DB to be persisting these.

Reply:
resolved